WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+/// Displays a Blog's User Details
+///
+class PersonViewController : UITableViewController {
+    
+    // MARK: - Public Properties
+    var person  : Person!
+    var blog    : Blog!
+    
+    
+    // MARK: - View Lifecyle Methods
+    override func viewDidLoad() {
+        assert(person != nil)
+        assert(blog != nil)
+        
+        super.viewDidLoad()
+        
+        title = person.fullName.nonEmptyString() ?? NSLocalizedString("Blog's User", comment: "Blog's User Profile. Displayed when the name is empty!")
+        WPStyleGuide.configureColorsForView(view, andTableView: tableView)
+    }
+    
+    
+    // MARK: - UITableView Methods
+    override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        tableView.deselectSelectedRowWithAnimation(true)
+        
+        let handlerMap = [
+            roleCell    : #selector(handleRoleWasPressed),
+            removeCell  : #selector(handleRemoveWasPressed)
+        ]
+
+        guard let cell = tableView.cellForRowAtIndexPath(indexPath), let handler = handlerMap[cell] else {
```

Comment:
To be honest the dictionary mapping is pretty smart / interesting, so if you really want to keep it, it's ok.

I'd suggest, however, using something simpler for the reader, especially since the mapping details don't need to be part of this method.

Why not use a method instead of a dictionary?

`guard let cell = tableView.cellForRowAtIndexPath(indexPath), let handler = handlerForCell(cell) else {`

It's not that the current code isn't right, it's just a tiny step towards making this method cleaner.  My 2 cents.